docs/explanation/gno-tooling/cli/gnokey.md

```diff
@@ -208,13 +208,17 @@ gnokey maketx call \
 
 #### **makeCallTx Options**
 
-| Name      | Type       | Description                      |
-| --------- | ---------- | -------------------------------- |
-| `send`    | String     | The amount of coins to send.     |
-| `pkgpath` | String     | The package path (required).     |
-| `func`    | String     | The contract to call (required). |
-| `args`    | String \[] | The arguments of the contract.   |
-
+| Name      | Type   | Description                               |
+| --------- | ------ |-------------------------------------------|
+| `send`    | String | The amount of coins to send.              |
+| `pkgpath` | String | The package path (required).              |
+| `func`    | String | The contract to call (required).          |
+| `args`    | String | An argument of the function being called. |
+
+:::info
```

Comment:
Sure, let's say we only support primitive types, uint8 slices and arrays, and link to [this](https://github.com/gnolang/gno/blob/5bc611abd48b4d514a7499116fec22d80fa80233/gno.land/pkg/sdk/vm/convert.go#L16).

Reply:
[2444f37](https://github.com/gnolang/gno/pull/1413/commits/2444f37ad99c1baaa0ee5878c9650a0740f0346f)

Reply:
Is this document manually created or auto-generated? I suggest it should be auto-generated, with our efforts aimed at improving the `--help` command. Essentially, your PR would modify a .go file, and we'd use a script (like `embedmd`?) to update this document accordingly.

Reply:
Instead of writing “Can be used multiple times”, I suggest we just use []String in the second column.

Or maybe StringList, StringArray, or whatever makes sense for such documentation.

Reply:
@moul 

I will do research on auto generation for this, but for now I would like to do this manually. Made a [draft issue](https://github.com/orgs/gnolang/projects/16/views/1?pane=issue&itemId=46763643) for this.

Previously, the second column had `String[]`, but I believe that is not accurate. If I was reading this for the first time trying to understand how the `-args` option works, based on the previous version, I would try calling it by
 `-args "arg1" "arg2" "arg3`" or similar, which wouldn't work. This is why I suggest we keep this change that I introduced, as it precisely defines the relationship between the option and its parameter.